src/test/javascript/spec/component/lecture/lecture-update.component.spec.ts

```diff
@@ -45,18 +51,22 @@ describe('LectureUpdateComponent', () => {
         pastLecture.endDate = yesterday;
 
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, FormsModule, MockModule(NgbTooltipModule)],
+            imports: [ArtemisTestModule, ArtemisSharedModule, FormsModule, MockModule(NgbTooltipModule), MockModule(OwlDateTimeModule)],
             declarations: [
                 LectureUpdateComponent,
-                MockComponent(LectureTitleChannelNameComponent),
+                LectureTitleChannelNameComponent,
```

Comment:
Can we mock some of those or are all required within the tests?

Reply:
We do not seem to need the ArtemisSharedModule import here, besides that I am not aware that we could optimize iz further.
I would assume that ngMocks still have problems with mocking signals, but I did not investigate it in this particular case.